### ISSUE TITLE
glitches in the v7.3 Alpha build

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#4225]
- [X] infinite scroll works fast and beautiful !!! Extremely responsive GUI experience.
- [x] upon startup the keyboard focus is not intuitive. "discovered channels" view has operational mousewheel scroll, but the "page-down" key does not work. Also, when doing keyword search, the keyboard focus stays there and <page-down> does not work when mouse is hovering across results.
- [x] the channels from converted v7.2 content are listed in v7.3. However, the "amount of torrents in channel" column is incorrect. Usually, only about half that amount of swarms are actually visible.
- [x] on scrolling, the mouse can be hidden. When clicking on an item and the "details box" appears, Tribler does not re-calculate the screen and scroll window. There are items hidden behind the details box. See the blue narrow row of pixels at bottom:
![screenshot from 2019-02-13 13-55-58](https://user-images.githubusercontent.com/325224/52713281-2a6f1180-2f97-11e9-8f15-b8f8300177e8.png)
- [x] doing a search or another action and navigating back to the "discovered channels" screen goes wrong sometimes. Then it shows this black screen, also after waiting for minutes:
![screenshot from 2019-02-13 13-59-49](https://user-images.githubusercontent.com/325224/52713622-17a90c80-2f98-11e9-97de-162f042c1ef9.png)
- [x] Same blank-screen bug also appears for search. Several OK searches, then this:
![screenshot from 2019-02-13 14-11-15](https://user-images.githubusercontent.com/325224/52714100-5be8dc80-2f99-11e9-8fa2-6cc9598554cf.png)
- [x] Again, occasionally eternal loading:
![screenshot from 2019-02-13 14-16-39](https://user-images.githubusercontent.com/325224/52714414-137dee80-2f9a-11e9-91c6-0235f3a6eec7.png)
- [x] Nasty errors in the logs of Traceback level
```--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:653:_runCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/endpoints.py:954:startConnectionAttempts
] http://tracker.Debian--bay.org/announce
INFO    1550062866.55                   session:264  (HttpTrackerSession)  Error when querying http tracker: [Failure instance: Traceback: <class 'twisted.internet.error.DNSLookupError'>: DNS lookup failed: no results for hostname lookup: tracker.Debian--bay.org.
/usr/lib/python2.7/dist-packages/twisted/internet/_resolver.py:137:deliverResults
/usr/lib/python2.7/dist-packages/twisted/internet/endpoints.py:900:resolutionComplete
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:459:callback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:567:_startRunCallbacks
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:653:_runCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/endpoints.py:954:startConnectionAttempts
] http://tracker.Debian--bay.org/announce
```

**Comment by ichorid** [comment]
Lock-ups problem is now solved in https://github.com/Tribler/tribler/pull/4090

**Comment by ichorid** [comment]
The traceback errors are unrelated to #4090  and should be considered in a separate issue. I'll create one.

**Comment by synctext** [comment]
Great! Any new Friday Alpha2 build and I'll do another intense test drive. Then with multiple computers and channel discovery+search.

**Comment by synctext** [comment]
Tried out the [fresh build on Jenkins](https://jenkins-ci.tribler.org/job/Build-Tribler_release/job/Build-Ubuntu64/174/)
- [X] When downloading through a tunnel, the torrent collecting is smart. We no longer try to download the torrent again when the torrent moves from the download pup-up to the download tab.
- [ ] Another issue with search results is the hidden scroll bar. There are 133 results (family filter OFF). If Tribler is enlarged that it can show over 50 rows, the scroll bar is omitted.
![screenshot from 2019-02-25 12-55-00](https://user-images.githubusercontent.com/325224/53335977-3c3ca700-38fd-11e9-863c-536c2e3d5da8.png)
- [ ] simple conversion error. Altering the default imported and converted setting goes wrong for the "0.0" bandwidth limit setting. Needs to be changed into "0" before saved settings are accepted
![screenshot from 2019-02-25 12-57-18](https://user-images.githubusercontent.com/325224/53336044-6b531880-38fd-11e9-9a4a-5050eb4be984.png)

**Comment by qstokkink** [comment]
Suddenly my channel got a duplicate of an existing torrent:

![afbeelding](https://user-images.githubusercontent.com/3630389/53338872-15826e80-3905-11e9-9d2f-10358bfd3455.png)

And when trying to delete this duplicate entry:

![afbeelding](https://user-images.githubusercontent.com/3630389/53338889-2337f400-3905-11e9-8b0c-5f5ef054924d.png)

**Comment by qstokkink** [comment]
I subscribed to this channel and the channel does not list 1 subscriber:

![afbeelding](https://user-images.githubusercontent.com/3630389/53339202-f506e400-3905-11e9-99ea-dc0f3cf675b0.png)

**Comment by ichorid** [comment]
@qstokkink , the subscribe counter is not supposed to be working until we finish the voting system. And I'm not sure if we will finish it before 7.3 release.

**Comment by ichorid** [comment]
Fixed by #4374
